README.md

```diff
@@ -15,30 +15,54 @@ This repository contains the frontend and backend code for CovEd's revamped webs
 ### 🐑 Install dependencies
 To install dependencies run the following.
 
-```
+```bash
 $ npm install
 $ npx lerna bootstrap
 ```
 
+#### Dependency Errors
+
+##### Not found in `package.json`
+Ensure that you install from the `/packages/client/` directory, NOT the root directory. All dependecies and packages should be added on to either `client` or `server`. 
+
+```bash
+$ npm uninstall <all new packages>
+$ cd packages/client
+$ npm install <all new packages>
+```
+
+##### `dev-dependecies`
+Ensure that the package is part of `dependencies` not `dev-dependecies`. To fix it:
+```bash
+$ npm uninstall <all new packages>
+$ npm install <all packages>
+```
+
 ### 🛠️ Build 
 
 #### Development Server
 This will start the live reload servers for both the backend api and the react client.
-```
+```bash
 $ npm start
 ```
 
 #### Production Mode
 This will build the client and start the server in production mode 
-```
+```bash
 $ npm run production
 ```
 
 ### 🧪Testing
 Run the following command to run all tests
-```
+```bash
 $ npm test
 ```
+#### Linting
+If there are a lot of errors regarding the linter, run the following command to run it locally 
+```bash
+npx lerna run lint
+```
+**Caveat:** Do NOT attempt to install `eslint` as a package as it will break the code. Use the above code, but install all necessary extensions (most code editors already support it).
```

Comment:
Is there a particular reason someone would try to install eslint?

Reply:
As of now, it is not included as a package. When one attempts to run 
```
eslint
```
 an error occurs; hence, one would attempt to remedy this by installing the eslint package. However, this is a breaking change and one should just run 
```
npx lerna run lint
``` 

**Inspiration**: Me :disappointed: